src/ngx_http_lua_pipe.h

```diff
@@ -80,8 +80,10 @@ typedef struct {
 } ngx_http_lua_pipe_signal_t;
 
 
+#ifndef NGX_LUA_NO_FFI_API
 void ngx_http_lua_pipe_init(void);
 ngx_int_t ngx_http_lua_pipe_add_signal_handler(ngx_cycle_t *cycle);
+#endif
```

Comment:
We still need these symbols to be defined when `-DNGX_LUA_NO_FFI_API` is passed to the compiler. As of now, this option would not compile. Probably best to conditionally implement those to be "NOP" when the FFI API is disabled?